content/docs/tasks/security/auth-sds/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Identity provision through SDS
+description: Shows how to enable SDS(secret discovery service) for istio identity provision.
+weight: 70
+keywords: [security,auth-sds]
+---
+
+This task shows how to enable [SDS(secret discovery service)](https://www.envoyproxy.io/docs/envoy/latest/configuration/secret#config-secret-discovery-service) for istio identity provision.
+
+Today the keys and certificates of Istio workloads are generated by Citadel and distributed to sidecars
+through secret-mount files, this approach has below side effects:
+* Traffic interruption:
+  When certificate rotation happens, envoy(pilot agent) needs to restart to pick up key/certificate update,
+  which causes traffic interruption.
+* Potential security issue:
+  Since key/certificate are file-mounted to sidecars, hackers could remote to sidecar container and steal key/certificate content.
+
+To solve the above issues, starting from 1.1 release, Istio supports identity provision through SDS(secret discovery service).
```

Comment:
These issues are addressed in Istio 1.1 through the support of SDS to provision identities.

Reply:
updated.